CMakeLists.txt

```diff
@@ -219,6 +219,9 @@ add_test(NAME cpp_clang_format COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/clang_format
 # Add test which checks all files have a valid license
 add_test(NAME reuse_lint COMMAND reuse lint)
 
+# Add test which checks that a build rerun doesn't compile and uses caches
+add_test(NAME cargo_build_rerun COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/check_cargo_build_rerun.sh" "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
I think there is the potential of a "race" condition here.

As if this is in the middle of running, another test might happen to be also compiling the `qml-minimal-no-cmake` example.

Is there a way we can force CTests to run a test after all the others have completed ? Or just run it by itself etc ?

Reply:
Glorious found that you supposedly do `set_tests_properties(test_name PROPERTIES RUN_SERIAL TRUE)`

Also set this on our `reuse` test as before we have had it where the book test generates new html files and reuse thinks they are unlicensed :-D